### PR TITLE
Emit deposited, exitStarted and finalizedExit events

### PR DESCRIPTION
**Pull request by tkmct**
I want these events emitted.

### CODE REVIEW

File: packages/wallet/src/wallet.ts [hunk 1]
```diff
@@ -187,13 +189,14 @@ export class ChamberWallet extends EventEmitter {
...
       console.log('Deposited', e)
-      this.handleDeposit(
+      const tx = this.handleDeposit(
         e.values._depositer,
         e.values._tokenId,
         e.values._start,
         e.values._end,
         e.values._blkNum
       )
+      this.emit('deposited', { wallet: this, tx })
```
Comment (syuhei176) [inline packages/wallet/src/wallet.ts]: Maybe you want call this in `handleDeposit`! When users deposit themselves, `handleDeposit ` was called from another function.

Comment (syuhei176) [reply packages/wallet/src/wallet.ts]: https://github.com/cryptoeconomicslab/plasma-chamber/blob/a083277095fa48302b981aeefab6731da2aa55b1/packages/wallet/src/wallet.ts#L438

Comment (tkmct) [reply packages/wallet/src/wallet.ts]: okay I'll change it.
thank you

**Comment by tkmct** [comment]
@syuhei176 I updated the place to call `this.emit` please check and approve if possible!

**Pull request merged**